db/seedCSVs.js

```diff
@@ -24,11 +27,11 @@ const insertBookingRecords = function () {
   const bookingRecords = seededData.generateBookings();
 
   const dataGen = () => {
-    const bookingsFile = fs.createWriteStream('bookings_data.csv');
-    for (let i = 0; i < 100000000; i += 1) {
```

Comment:
You could consider passing in a variable to the function that defines your data generation count. That way, if you ever want to scale the function to produce a smaller file for testing, you don't have to change anything about the generic function itself, for loops, end-loop logic, etc. Just invoke the function with a new, smaller value.